### DISCUSSION TITLE
How do I pause sync service?

### DISCUSSION THREAD

**Discussion by kokoajin** [abraunegg/onedrive#1440] [Q&A] [answered]
How do i pause and resume sync or monitor service for onedrive?

I do not see any option to pause and resume service in USAGE.md.

**Comment by abraunegg** [comment, marked as answer]
@kokoajin 
There is no 'pause' or 'resume'

To do this, 'pause' is 'stop', resume is 'start'

**Reply by kokoajin** [reply]
Thank you for your answer.
But I new to linux, so when you mean "stop", I just have to stop the service by `systemctl --user stop onedrive` right ?

**Reply by abraunegg** [reply]
@kokoajin 
Correct - this will stop that particular service
